mlir/lib/Dialect/Bufferization/IR/BufferizationDialect.cpp

```diff
@@ -63,6 +64,8 @@ void mlir::bufferization::BufferizationDialect::initialize() {
 #include "mlir/Dialect/Bufferization/IR/BufferizationOps.cpp.inc"
       >();
   addInterfaces<BufferizationInlinerInterface>();
+  declarePromisedInterfaces<BufferizableOpInterface, func::CallOp, func::FuncOp,
+                            func::ReturnOp>();
```

Comment:
This should be in the Func dialect I believe

Reply:
Just to confirm, is [this reasoning](https://github.com/llvm/llvm-project/pull/78368#discussion_r1475542616) incorrect?

Reply:
[Move Func Op declared interfaces to the Func Dialect code.](https://github.com/llvm/llvm-project/pull/78368/commits/8273ef4fe34d030c4ec9e1d5ea9426502e4451f1) should address this.